### PR TITLE
3 feature 3 graph system

### PR DESCRIPTION
**Pull request by snkrs23**

### CODE REVIEW

File: src/main/java/com/playground/playground/GraphSystemController.java
Comment (alvina-yang) [inline src/main/java/com/playground/playground/GraphSystemController.java:1]: I think there might be some commented code in here

File: src/main/resources/com/playground/playground/graph-system-view.fxml
Comment (alvina-yang) [inline src/main/resources/com/playground/playground/graph-system-view.fxml:1]: I think that the xAxis and yAxis ids are swapped in this file but other than that everything looks good!

File: src/main/resources/com/playground/playground/graph-system-view.fxml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<?import javafx.scene.control.CheckBox?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+
+<VBox prefHeight="457.0" prefWidth="485.0" xmlns="http://javafx.com/javafx/20.0.1" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.playground.playground.GraphSystemController">
+   <VBox prefHeight="462.0" prefWidth="485.0">
+      <children>
+          <ScatterChart fx:id="neuralNetwork" maxHeight="-Infinity" maxWidth="-Infinity" prefHeight="400.0" prefWidth="400.0" styleClass="neural-network-text">
+              <xAxis>
+                  <NumberAxis fx:id="yAxis" lowerBound="-5.5" side="BOTTOM" upperBound="5.5" />
```
Comment (Shivesh777) [inline src/main/resources/com/playground/playground/graph-system-view.fxml:15]: The bounds for the axes should be from -10 to 10 to be consistent with our dataset.
```suggestion
                  <NumberAxis fx:id="yAxis" lowerBound="-10" side="BOTTOM" upperBound="10" />
```

File: src/main/resources/com/playground/playground/graph-system-view.fxml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+              </xAxis>
+              <yAxis>
+                  <NumberAxis lowerBound="-5.5" side="RIGHT" upperBound="5.5" fx:id="xAxis" />
```
Comment (Shivesh777) [inline src/main/resources/com/playground/playground/graph-system-view.fxml:18]: The bounds for the axes should be from -10 to 10 to be consistent with our dataset here too.
```suggestion
                  <NumberAxis lowerBound="-10" side="RIGHT" upperBound="10" fx:id="xAxis" />
```

File: src/main/java/com/playground/playground/GraphSystemController.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+     * the root object was not localized.
+     */
+    public void initialize(URL location, ResourceBundle resources) {
+//        XYChart.Series<String, Double> seriesHigh = new XYChart.Series<>();
+//        seriesHigh.getData().add(new XYChart.Data<>("SubLabel1", 20.9));
+//        seriesHigh.getData().add(new XYChart.Data<>("SubLabel2", 30.9));
+//        seriesHigh.getData().add(new XYChart.Data<>("SubLabel3", 40.9));
+//
+//        XYChart.Series<String, Double> seriesLow = new XYChart.Series<>();
+//        seriesLow.setName("Label2");
```
Comment (Shivesh777) [inline src/main/java/com/playground/playground/GraphSystemController.java:37]: This function seems to be unimplemented, was it intended to be so? If so, could you please remove the commented code.

**Review by Shivesh777 — CHANGES_REQUESTED** [review]

File: src/main/resources/com/playground/playground/graph-system-view.fxml (hunk 1 above)
Comment (snkrs23) [reply src/main/resources/com/playground/playground/graph-system-view.fxml:15]: I'll change that.

File: src/main/java/com/playground/playground/GraphSystemController.java (hunk 3 above)
Comment (snkrs23) [reply src/main/java/com/playground/playground/GraphSystemController.java:37]: The code is there for reference when implementing the graphing feature with the front end implementation.

File: src/main/resources/com/playground/playground/graph-system-view.fxml (hunk 2 above)
Comment (snkrs23) [reply src/main/resources/com/playground/playground/graph-system-view.fxml:18]: I'll change that.

**Review by Shivesh777 — APPROVED** [review]

File: .idea/.gitignore [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (Rishit-dagli) [inline .idea/.gitignore:1]: You should revert all .idea changes

**Review by Rishit-dagli — CHANGES_REQUESTED** [review]

**Review by Rishit-dagli — COMMENTED** [review]
Gradle builds fail and there are also merge conflicts, could you take a look at those

**Comment by snkrs23** [comment]
> Gradle builds fail and there are also merge conflicts, could you take a look at those

We cannot make any changes as mentioned in the group chat until the PR is approved.

**Comment by Rishit-dagli** [comment]
Doesn't look like this is ready for review, there are merge conflicts and the gradle build fails

**Review by alvina-yang — APPROVED** [review]

**Pull request closed without merging**